src/features/textEditor/components/tipTapNodes/references/ReferencesList.tsx

```diff
@@ -9,10 +9,9 @@ import { usePopper } from 'react-popper';
 
 import { getReferencesAtom } from '../../../../../atoms/referencesState';
 import { cx } from '../../../../../lib/cx';
-import { ReferenceItem } from '../../../../../types/ReferenceItem';
 import { fuseOptions } from './config';
 
-export type ReferenceListProps = SuggestionProps<{ id: string; label: string }>;
+export type ReferenceListProps = SuggestionProps<{ id: string }>;
 
 export const ReferencesList = forwardRef((props: ReferenceListProps, ref) => {
```

Comment:
What do you think of renaming this as we already have one named like this one on the references sidebar.
Maybe `ReferencesListPopup`